src/features/HelpAndKnowledge/Articles/StartingMTOMatrix/index.tsx

```diff
@@ -0,0 +1,161 @@
+  const { t } = useTranslation('startingMto');
+
+  const step4List = tArray<string>('startingMto:step4.list');
+
+  return (
+    <div>
+      <MainContent>
+        <GridContainer>
+          <Grid tablet={{ col: 12 }}>
+            <HelpBreadcrumb text={t('breadcrumb')} />
+
+            <div className="mint-body-medium">
+              <PageHeading className="line-height-large margin-bottom-2 margin-top-4">
+                {t('title')}
+              </PageHeading>
+
+              <HelpCategoryTag type={ArticleCategories.MTO_TUTORIALS} />
+
+              <p className="mint-body-large margin-top-1 margin-bottom-4">
+                {t('description')}
+              </p>
+
+              <ProcessList>
+                {/* STEP 1 */}
+                <ProcessListItem className="maxw-none">
+                  <ProcessListHeading type="h3">
+                    {t('step1.heading')}
+                  </ProcessListHeading>
+                  <p>
+                    <Trans
+                      t={t}
+                      i18nKey="step1.text"
+                      components={{
+                        link1: (
+                          <Link
+                            href="https://mint.cms.gov"
+                            target="_blank"
+                            variant="external"
+                          >
+                            {' '}
+                          </Link>
+                        )
+                      }}
+                    />
+                  </p>
+                </ProcessListItem>
+
+                {/* STEP 2 */}
+                <ProcessListItem className="maxw-none">
+                  <ProcessListHeading type="h3">
+                    {t('step2.heading')}
+                  </ProcessListHeading>
+
+                  <p>{t('step2.text')}</p>
+
+                  <img
+                    alt={t('step2.altText')}
+                    src={MINTHome}
+                    className="border-1px border-base-lightest radius-md shadow-2"
+                  />
+
+                  <p>{t('step2.caption')}</p>
+                </ProcessListItem>
+
+                {/* STEP 3 */}
+                <ProcessListItem className="maxw-none">
+                  <ProcessListHeading type="h3">
+                    {t('step3.heading')}
+                  </ProcessListHeading>
+
+                  <p>{t('step3.text')}</p>
+
+                  <img
+                    alt={t('step3.altText')}
+                    src={collaborationAreaNotStarted}
+                    className="border-1px border-base-lightest radius-md shadow-2"
+                  />
+
+                  <p>{t('step3.caption')}</p>
+                </ProcessListItem>
+
+                {/* STEP 4 */}
+                <ProcessListItem className="maxw-none">
+                  <ProcessListHeading type="h3">
+                    {t('step4.heading')}
+                  </ProcessListHeading>
+
+                  <p>{t('step4.text')}</p>
+
```

Comment:
The gap between text and list seems bigger than figma
<img width="439" alt="Screenshot 2025-05-22 at 11 33 15 AM" src="https://github.com/user-attachments/assets/7ae35da9-d56a-4320-9fc0-8553a6898a76" />